IM_calculation/IM/intensity_measures.py

```diff
@@ -58,10 +58,14 @@ def get_SDI_nd(acceleration, period, NT, DT, z, alpha, dy, dt):
     if acceleration.ndim != 1:
         ts, dims = acceleration.shape
         Nstep = calculate_Nstep(DT, NT)
```

Comment:
Is Nstep needed anymore?

Reply:
Good point - please review my latest change